test/basic.test.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import ReactDOM from 'react-dom';
+import * as Basic from './cases/basic';
```

Comment:
I feel that splitting these tests into two files is a bit disconnected. I think it's better to combine Components and tests into a single file.